api/v1beta2/error.go

```diff
@@ -0,0 +1,11 @@
+package v1beta2
+
+import "errors"
+
+var (
+	ErrTypeAssertKyma           = errors.New("raw can not be cast to v1beta2.Kyma")
```

Comment:
I don't understand what `raw` means in the generic context. Perhaps the `value can not be cast to v1beta2.Manifest` would be better?